logic/strategy/calculated_peg.go

```diff
@@ -6,6 +6,11 @@ import (
 )
 
 func PegHighestCardNow(hand []model.Card, prevPegs []model.PeggedCard, curPeg int) (model.Card, bool) {
+	// TODO maybe be smarter when the count is zero (i.e. don't play a 5, but play as high a card as possible)
```

Comment:
"highest card" here, doesn't mean "highest pegging value" but "card which gets me the most points immediately". so I don't think we need to worry about that:#

Reply:
Yep, but I just meant that my strategy when playing cribbage would be to peg my highest card if there's nothing on the board yet... for now I will leave it as just choosing the first card in the hand though